package.json

```diff
@@ -46,7 +46,7 @@
     "@typescript-eslint/eslint-plugin": "^8.7.0",
     "@typescript-eslint/parser": "^8.7.0",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-import-x": "^3.1.0",
+    "eslint-plugin-import-x": "^4.4.3",
```

Comment:
Major changes of v3 to v4
- Use typescript-eslint v8. 
- The minimum supported ESLint version is now >= 8.57.0 
- minimum required Node.js version is now 18.18.0.

c.f.
- https://github.com/un-ts/eslint-plugin-import-x/releases/tag/v4.0.0